src/components/EarlyAccessRegistrySurvey.tsx

```diff
@@ -0,0 +1,49 @@
+import { useEffect, useState } from "react";
+import DynamicSurvey from "./DynamicSurvey";
+import { Survey } from "posthog-js";
+import { usePostHog } from "posthog-js/react";
+
+// TODO: Make this an ENV variable
+const SURVEY_ID = "019683f6-4fe2-0000-d182-6ef8f3982fc3";
+
+export const EarlyAccessRegistrySurvey = () => {
+  const posthog = usePostHog();
+  const [surveyData, setSurveyData] = useState<Survey | null>(null);
+
+  function loadSurvey() {
+    posthog.getSurveys((surveys) => {
+      const survey = surveys.find((s) => s.id === SURVEY_ID);
+
+      if (survey) {
+        setSurveyData(survey);
+        posthog?.capture("Early Access Registry Survey viewed", {
```

Comment:
I don't think we need this event. We wanted an explicit one for the registration survey given how sensitive the sign up flow UX is. But we should be able to leverage the generic `survey sent` event for this one I think. 

LMK if you think differently!

Reply:
removed in 57e5a2ea2b28c30bd60aeaeca36e6edfd7744d8f